bazel/repository_locations.bzl

```diff
@@ -359,10 +359,10 @@ DEPENDENCY_REPOSITORIES = dict(
         use_category = ["build"],
     ),
     rules_foreign_cc = dict(
-        sha256 = "3184c244b32e65637a74213fc448964b687390eeeca42a36286f874c046bba15",
-        strip_prefix = "rules_foreign_cc-7bc4be735b0560289f6b86ab6136ee25d20b65b7",
-        # 2019-09-26
-        urls = ["https://github.com/bazelbuild/rules_foreign_cc/archive/7bc4be735b0560289f6b86ab6136ee25d20b65b7.tar.gz"],
+        sha256 = "655c02d272e95af0ff4afe805ffa3768b04d14fb50e92d5e25e9f12f5f15a681",
```

Comment:
81fa89d2caac59d9770605084d24f7bcc74cb8cc28d4019c89152fb4a09e5d15

for the new sha256

the new rule is https://github.com/bazelbuild/rules_foreign_cc/archive/f54b7ae56dcf1b81bcafed3a08d58fc08ac095a7.zip